index.rst

```diff
@@ -379,52 +355,50 @@ In order to merge a branch to the develop branch, each job must:
       It does not check format/readability/sensible inputs etc.
 
 
-No integration testing (on the ncsa-teststand) is strictly required, however, one would hope that the script has run
+No integration testing (on the NCSA-teststand) is strictly required, however, one would hope that the script has run
 successfully through the integration-test-stand or on the summit.
 
 
-Jobs in Production
-^^^^^^^^^^^^^^^^^^
+Scripts in Production
+^^^^^^^^^^^^^^^^^^^^^
 
-Jobs in full production are to be kept in the `ts_queueJobs` repository. This is the last step in the
-development process. Jobs in this category are tightly controlled and standards are strictly enforced. No production
-level Job can call any utility in the `Observing Utilities`_ repository. All utilities must be sanctioned Control
-Package Utilities. All jobs in this repository must be runnable at all times by any operator. All code
-requires documentation to a level where other developers can diagnose the
-code and fix any issues that are resulting in failed tests. This must include a description of the Job, a
+Scripts in full production are to be kept in the `ts_scripts` repository. This is the last step in the
+development process. Scripts in this category are tightly controlled and standards are strictly enforced. No production
+level script can call any utility in the `Observing Utilities`_ repository. All called utilities shall be sanctioned
+Control Package Utilities. All scripts in this repository shall be runnable at all times by any operator. All code
+shall be documented at a level where other developers can diagnose the
+code and fix any issues that are resulting in failed tests. This shall include a description of the script, a
 description of the inputs/outputs, and depending on the complexity of the function an example may be required.
-All required metadata for the script must be accurate (e.g. completion times).
-
-.. note::
-
-    The ts_standardscripts repo currently holds the production Jobs but will be renamed.
+All required metadata for the script shall be accurate (e.g. completion times). The following testing requirements shall
+also be met.
 
 
 Required Testing
 ----------------
 
-In order to merge to develop the following level of testing must be implemented and passing:
+In order to merge to develop the following level of testing shall be implemented and passing:
 
-- Code must be fully documented.
+- Code shall be fully documented.
 
 - Have (and pass) a unit test showing the script is of a format that is capable of being executed
 
-    - This will use the helper class in standardscripts already (BasescriptTestCase). This verifies the
+    - This will use the helper class in ts_scripts (currently ts_standardscripts) already (BasescriptTestCase).
```

Comment:
Suggestion:

> This can be accomplished by using the BasescriptTestCase helper class already available in ts_scripts (currently ts_standardscripts).

Reply:
Incorporated... but re-worded a tiny bit.